source/component/pasdoc.css.inc

```diff
@@ -122,6 +122,20 @@
 'div.nodescription { color:red; }' + LineEnding +
 'dl.parameters dt { color:blue; }' + LineEnding +
 '' + LineEnding +
+'/* style for warning and note tag */' + LineEnding +
```

Comment:
The changes should be done by editing  `pasdoc.css`, and then the `pasdoc.css.inc` is auto-generated by just running `make` in source/component/ subdirectory. This `make` invocation (following `source/component/Makefile` instructions) will run our own tool `file_to_pascal_string`.

So, first you need to compile `file_to_pascal_string` :) The easiest way to do it i execute `make build-tools` in the top-level PasDoc directory.